en/live/global.ini

```diff
@@ -67982,8 +67982,8 @@ ui_controlhint_moboglas_open_map=mobiGlas (Map)
 ui_controlhint_move_to_exit=Move To Exit
 ui_controlhint_navigate=Navigate / Select
 ui_controlhint_or=Or
-ui_controlhint_pc_choice:interaction_mode=Interact
-ui_controlhint_pc_choice:interaction_system=Interact Mode
+ui_controlhint_pc_choice_interaction_mode=Interact
```

Comment:
Nachfolgend auch. Wie gesagt, bin ich dagegen weil wir nicht wissen ob das tatsächlich ein Fehler ist. Ich würde es einfach der ini Datei ans Ende anfügen. Wie es bereits mit `itemPort_port_NameMissileRack0=Raketengestell` umgesetzt wurde

Reply:
Die sind doch in der englischen auch so drin. Das ist kein Fehler. Oder ich Check einfach nicht was genau du da jetzt ändern willst

Reply:
Ach du meinst das Script mault dann rum?

Reply:
Nur der Vollständigkeit. Hab mal ChatGPT gefragt :D

> Ich habe eine ini datei. In dieser sind in den Variablen : Doppelpunkte enthalten. Was meinst du, ist das ein Fehler?
> 
> ui_controlhint_moboglas_open_map=mobiGlas (Karte)
> ui_controlhint_move_to_exit=Zum Ausgang bewegen
> ui_controlhint_navigate=Navigieren/Auswählen
> ui_controlhint_or=Oder
> ui_controlhint_pc_choice:interaction_mode=Interagieren
> ui_controlhint_pc_choice:interaction_system=Interaktionsmodus
> ui_controlhint_pc_drink_glug=Trinken
> ui_controlhint_pc_drink_sip=Nippen
> ui_controlhint_pc_food=Essen
> 
> ChatGPT
> Nein, es ist kein Fehler. In INI-Dateien können Doppelpunkte verwendet werden, um Werte zu trennen oder Schlüssel zu benennen. In deinem Beispiel scheinen die Doppelpunkte dazu zu dienen, verschiedene Optionen oder Modi innerhalb der Steuerungshinweise zu kennzeichnen. Solange die INI-Datei korrekt strukturiert ist und von der Anwendung richtig interpretiert wird, sollten die Doppelpunkte in diesem Kontext in Ordnung sein.